rotkehlchen/api/rest.py

```diff
@@ -3929,9 +3929,7 @@ def add_manual_latest_price(
                 result=wrap_in_fail_result(message=str(e)),
                 status_code=HTTPStatus.CONFLICT,
             )
-        # also invalidate the usd price equivalent of the `from_asset`
-        pairs_to_invalidate.append((from_asset, A_USD))
-        self._invalidate_asset_pairs_prices(pairs_to_invalidate)
```

Comment:
Don't you need to invalidate USD still?

Reply:
With the current approach, there's no need to manually invalidate USD price.